src/observer.ts

```diff
@@ -362,21 +398,26 @@ export class DataLayerObserver {
 
     try {
       const target = DataLayerTarget.find(source);
```

Comment:
The `DataLayerTarget.find` no longer throws.  It's still in a try catch for unexpected issues, but it no longer throws for situations where the data layer is `undefined`.